src/codegen/function.rs

```diff
@@ -74,6 +75,12 @@ pub fn generate(
     version_condition(w, env, analysis.version, commented, indent)?;
     not_version_condition(w, analysis.not_version, commented, indent)?;
     doc_hidden(w, analysis.doc_hidden, comment_prefix, indent)?;
+    if !in_trait || only_declaration {
+        doc_alias(w, &analysis.glib_name, comment_prefix, indent)?;
+        if analysis.new_name.is_some() {
+            doc_alias(w, &analysis.name, comment_prefix, indent)?;
```

Comment:
An example for this second doc_alias:

```rust
#[doc(alias = "gdk_pixbuf_new_from_bytes")]
#[doc(alias = "new_from_bytes")]
pub fn from_bytes() {}
```

I think there will be a lot of of equivalents for `new_from_bytes` and I'm sure there is much point into having it...

Reply:
Yes indeed. See new version.

Reply:
Looks much better, thanks!